addStyles.js

```diff
@@ -143,6 +143,20 @@ function createLinkElement(options) {
 function addStyle(obj, options) {
 	var styleElement, update, remove;
 
+	// If a transformCssOnLoad function was defined, run it on the css
+	if (options.transformCssOnLoad && obj.css) {
+		var transformationFunction = self[options.transformCssOnLoad];
+		if (!transformationFunction) {
+			throw new Error("A function named " + options.transformCssOnLoad + " needs to be available on the global scope.");
```

Comment:
A function on global scope seems dirty. Is it possible to pass a function instead of a name here?